aten/src/ATen/native/native_functions.yaml

```diff
@@ -6774,6 +6774,16 @@
 - func: sort.dimname(Tensor self, Dimname dim, bool descending=False) -> (Tensor values, Tensor indices)
   variants: method, function
 
+- func: msort.values(Tensor self, bool descending=False, *, Tensor(a!) values, Tensor(b!) indices) -> (Tensor(a!) values, Tensor(b!) indices)
```

Comment:
These look good but see comments above for tweaks.

Reply:
Addressed!